utilities/cli/holohub.py

```diff
@@ -1025,6 +1075,133 @@ def handle_clear_cache(self, args: argparse.Namespace) -> None:
+        except Exception as e:
+            print(Color.red(f"Failed to add application to applications/CMakeLists.txt: {str(e)}"))
+            print(Color.red("Please add the application manually to applications/CMakeLists.txt"))
+
+    def handle_create(self, args: argparse.Namespace) -> None:
+        """Handle create command"""
+        # Ensure template directory exists
+        template_dir = self.HOLOHUB_ROOT / args.template
+        if not template_dir.exists() and not args.dryrun:
+            holohub_cli_util.fatal(f"Template directory {template_dir} does not exist")
+
+        if not args.directory.exists() and not args.dryrun:
+            holohub_cli_util.fatal(f"Project output directory {args.directory} does not exist")
+
+        context = {
+            "project_name": args.project,
+            "project_slug": args.project,
+            "full_name": os.getenv("USER", "HoloHub User"),
+            "affiliation": "Holoscan",
+            "language": args.language,
+            "version": "0.1.0",
+            "holoscan_version": base_sdk_version,
+            "platforms": '["x86_64", "aarch64"]',
+            "tags": '["keyword1", "keyword2", "keyword3"]',
+            "description": "A brief description of the project",
+            "license": "Apache-2.0",
+            "year": datetime.datetime.now().year,
+        }
+
+        # Add any additional context variables from command line
+        if args.context:
+            for ctx_var in args.context:
+                try:
+                    key, value = ctx_var.split("=", 1)
+                    context[key] = value
+                except ValueError:
+                    holohub_cli_util.fatal(
+                        f"Invalid context variable format: {ctx_var}. Expected key=value"
+                    )
+        canonical_project_name = context["project_name"].lower().replace(" ", "_")
+        context["project_slug"] = canonical_project_name
+        project_dir = args.directory / canonical_project_name
+
+        # Print summary if dryrun
+        if args.dryrun:
+            print(Color.green("Would create project folder with these parameters (dryrun):"))
+            print(f"Directory: {project_dir}")
+            for key, value in context.items():
+                print(f"  {key}: {value}")
+            if args.directory == self.HOLOHUB_ROOT / "applications":
+                print(Color.green("Would modify `applications/CMakeLists.txt`: "))
+                print(f"    add_holohub_application({canonical_project_name})")
+            return
+
+        try:
+            import cookiecutter.main
+        except ImportError:
+            self._install_template_deps(args.dryrun)
+
+        import cookiecutter.main
+
+        if project_dir.exists():
+            holohub_cli_util.fatal(f"Project directory {project_dir} already exists")
+
+        try:
+            cookiecutter.main.cookiecutter(
+                str(template_dir),
+                no_input=True,
```

Comment:
Seems like `__prompts__` is not respected unless input is allowed (could control with a command parameter?)

```suggestion
                no_input=False,
```

Reply:
Yes, added --interactive or -i, default to true.